docs/start/setup-channel.mdx

```diff
@@ -165,8 +165,73 @@ print(result)
     from qiskit_ibm_runtime import QiskitRuntimeService
 
     service = QiskitRuntimeService()
+  ```
+
+## Access IBM Quantum Systems using REST API
+
+Alternatively, you can also access IBM Quantum Systems using REST APIs, enabling you to work with quantum systems from any programming language or framework. 
+
+1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account) and save it to a file named `token`.
```

Comment:
I think instead of step 1 I think you should copy the Step 1, 2 and 3 of the "Set up to use IBM Quantum Platform" section. This will keep things consistent

Reply:
Step 1 from "Set up to use IBM Quantum Platform" does not apply in the case of REST API since the whole idea is to access IBM Quantum Systems in a language/framework agnostic manner. We use python just for the sake of demonstrating API requests (which can be done with other languages/frameworks too). 

I've included Steps 2 &3.